### ISSUE TITLE
Plot empty

### ISSUE THREAD

**Issue by jessydit** [rhysf/Synima#31]
Hello, 
I would like to plot the synteny between 3 bacterial genomes, so each genome corresponds to a single contig and I would like to visualize the synteny between the different genes on these contigs. 
I ran the entire pipeline for the first time, without apparent problems. I used Orthofinder, which identified over 900 orthogroups with all three genomes present. However, at the end I obtain an almost empty plot, showing only the lengths of the 3 contigs. 
I attach the dagchainer coordinates files, the config.txt for the plot and the pdf. 

Any ideas where I should troubleshoot would be much appreciated! 

Cheers,
Jessica
[config.txt](https://github.com/rhysf/Synima/files/8264191/config.txt)
[config.txt.pdf](https://github.com/rhysf/Synima/files/8264192/config.txt.pdf)
[Dagchainer.zip](https://github.com/rhysf/Synima/files/8264214/Dagchainer.zip)

**Comment by rhysf** [comment]
hi @jessydit 

Thanks for opening this issue.

The dagchainer files look fine from a quick inspection, so it looks like the orthology and synteny prediction pipeline parts have worked correctly, and it's just the final visualization step that has not worked.

Can you provide the full command you used and all the output it provided?

In order to recreate the empty plot (assuming the output warnings doesn't include any clear issue), I will likely need the  the genome and peptide fasta and GFF files for each of the genomes as well - which you can attach here, or email if you would prefer them to remain confidential.

Best,
Rhys

**Comment by rhysf** [comment]
Hi @jessydit 

Thanks for this. There appear to be no issues in the screenshot, so it would be good for me to try and recreate the issue, which i can then try and figure out the issue.

If you can email me the GFF's, genome.fasta and annotation.pep and annotation.cds files, then that should be everything i need.

Thanks,
Rhys

**Comment by jessydit** [comment]
Great, thanks! Could you send me the correct email address to use please? 

Cheers,
Jessica

**Comment by rhysf** [comment]
rfarrer at broadinstitute dot org

**Comment by rhysf** [comment]
Hi @jessydit 

I've found the issue. Your genome.fa ID's say one thing, and the DAGchainer results have them listed as something else. Specifically:

Hepatincola_Av/Hepatincola_Av.genome.fa (fasta id = Candidatus Hepatincola isopodorum strain AV), but DAGchainer named the ID HAV

Hepatincola_Pdp/Hepatincola_Pdp.genome.fa (fasta id = Candidatus Hepatincola isopodorum strain Pdp), but DAGchainer named the ID HPDP

Hepatincola_Pp/Hepatincola_Pp.genome.fa )fasta id = Hepatincola isopodorum strain PP), but DAGchainer named the ID PP

By deleting the Synima output, and changing those genome.fa ID's, it correctly generated the figure.

I'd be keen to know if you changed those id's during the pipeline, or how DAGchainer had a different name to the ones listed in the fasta, in order to determine if i need to change the code?

I will reply to your email with the output and updated files.

Best,
Rhys

**Comment by jessydit** [comment]
Dear Rhys, 

awesome, thank you so much for finding the fix! I changed the genome names, ran it again and it worked like a marvel! 

I definitely did not change anything in the genome files during the pipeline but I do recall that I changed the names in the gff3 files at the very beginning because the candidatus names from the genome files had been truncated in the gff3 files. I was not aware that the fasta header in the genome files had to be identical as well. Thanks a lot for your help! 

All the best, 
Jessica
